src/Features/Core/Portable/Completion/Providers/TypeImportCompletionItem.cs

```diff
@@ -46,10 +47,41 @@ public static CompletionItem Create(INamedTypeSymbol typeSymbol, string containi
                  tags: GlyphTags.GetTags(typeSymbol.GetGlyph()),
                  rules: CompletionItemRules.Default,
                  displayTextPrefix: null,
-                 displayTextSuffix: typeSymbol.Arity == 0 ? string.Empty : "<>",
+                 displayTextSuffix: typeSymbol.Arity == 0 ? string.Empty : GetDisplayTextSuffixForGeneric(isCSharp),
                  inlineDescription: containingNamespace);
         }
 
+        public static CompletionItem CreateAttributeNameItem(CompletionItem attributeItem, string attributeName)
```

Comment:
Should this be called `attributeNameWithoutSuffix` or something? Or a doc comment describing what the expectation is.

Reply:
fixed